package.json

```diff
@@ -1171,6 +1173,16 @@
                     "command": "aws.apprunner.deleteService",
                     "group": "3@1",
                     "when": "view == aws.explorer && viewItem =~ /awsAppRunnerServiceNode.[RUNNING|PAUSED|CREATE_FAILED]/"
+                },
+                {
+                    "command": "aws.cloudFormation.newTemplate",
+                    "group": "0@1",
+                    "when": "view == aws.explorer && viewItem == awsCloudFormationRootNode"
```

Comment:
should we also enable it in the vscode file explorer on folders? that avoids the "save as..." dialog.

Reply:
I think we shouldn't add this to the file explorer. It doesn't scale.